app-client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "app-client",
   "homepage":
-    "https://s3.amazonaws.com/s3-bucketname-serverless-graphql-apollo",
+    "https://s3.amazonaws.com/emc-school-s3-serverless-graphql-apollo",
```

Comment:
this change is not required. idea is that anyone using this repo can use their own bucketname. long term we would want to use serverless plugin to generate random and unique bucket name.